x-pack/test/alerting_api_integration/security_and_spaces/tests/alerting/delete.ts

```diff
@@ -301,57 +302,59 @@ export default function createDeleteTests({ getService }: FtrProviderContext) {
             .send(getTestAlertData())
             .expect(200);
 
-          await supertest
-            .put(
-              `${getUrlPrefix(space.id)}/api/alerts_fixture/saved_object/alert/${createdAlert.id}`
-            )
-            .set('kbn-xsrf', 'foo')
-            .send({
-              attributes: {
-                name: 'bar',
-              },
-            })
-            .expect(200);
+          await retry.try(async () => {
```

Comment:
As discussed offline, the `retry.try` can wrap just the following `await supertest` call in each test.

Reply:
Updated!